tests/unit/Directors.spec.ts

```diff
@@ -105,13 +106,8 @@ describe('Directors as a COOP', () => {
     const instance = new Constructor({ store, vuetify })
     vm = instance.$mount()
 
-    // set as-of date
+    // set As Of Date to trigger watcher
     vm.asOfDate = '2019-04-01'
-
-    // set vm2 for draft test (setup is different and does not fire the 'asOfDate' watcher)
```

Comment:
This appears to be some work-around unit test. The As Of Date watcher is always fired (for new or draft COD) so this unit test is obsolete.